src/main/java/io/kestra/plugin/surrealdb/Query.java

```diff
@@ -0,0 +1,120 @@
+package io.kestra.plugin.surrealdb;
+
+import com.surrealdb.driver.SyncSurrealDriver;
+import com.surrealdb.driver.model.QueryResult;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.common.FetchType;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.serializers.FileSerde;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.io.*;
+import java.net.URI;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(title = "Query a Surreal database with SurrealQL.")
+@Plugin(
+		examples = {
+				@Example(
+						title = "Send a SurrealQL query to a Surreal database",
+						code = {
+								"useTls: true",
+								"port: 8000",
+								"host: localhost",
+								"username: surreal_user",
+								"password: surreal_passwd",
+								"database: surreal_db",
+								"namespace: surreal_namespace",
+								"query: SELECT * FROM SURREAL_TABLE",
+								"fetchType: STORE"
+						}
+				)
+		}
+)
+public class Query extends SurrealDBConnection implements RunnableTask<Query.Output>, QueryInterface {
+
+	@NotNull
+	@Builder.Default
+	protected FetchType fetchType = FetchType.STORE;
+
+	@Builder.Default
+	protected Map<String, String> parameters = new HashMap<>();
+
+	@NotNull
+	@NotBlank
+	protected String query;
+
+	@Override
+	public Query.Output run(RunContext runContext) throws Exception {
+		SyncSurrealDriver driver = super.connect(runContext);
+
+		String renderedQuery = runContext.render(query);
+
+		List<QueryResult<Object>> results = driver.query(renderedQuery, parameters, Object.class);
+
+		Query.Output.OutputBuilder outputBuilder = Output.builder().size(results.stream()
+				.mapToLong(result -> (long) result.getResult().size())
+				.sum());
```

Comment:
I'm not sure I understand the Surreal DB result model here, isn't results the list of results or is it a list of list of results?
If it's a list of results we should set the size to results.size().

Reply:
Surreal DB method query returns List of Wrappers which includes List<T> result, status and time. I guess is needed when one query executes on different models/tables. 
So if execute query, for example "SELECT * FROM TABLE", from one table, then it will return List<QueryResult> with one item (size 1), and inside this item will be List of result from this table (which can be more than one, so i not quite sure how to show size of rows correctly)